extensions/api/data-management/transferprocess/src/test/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/transform/DataRequestToDataRequestDtoTransformerTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.api.datamanagement.transferprocess.transform;
+
+import org.eclipse.dataspaceconnector.api.datamanagement.transferprocess.model.DataRequestDto;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataRequest;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DataRequestToDataRequestDtoTransformerTest {
+    DataRequestTransformerTestData data = new DataRequestTransformerTestData();
```

Comment:
Isn't that a standard approach? https://martinfowler.com/bliki/ObjectMother.html

Reply:
only because it's mentioned on fowler's blog does not mean that it's a "standard approach", especially for pages that are that old.
Anyway at the end of that page I read:

> Object Mothers do have their faults. In particular there's a heavy coupling in that many tests will depend on the exact data in the mothers. As a result it's tricky should you want to change that standard data for any reason. Changes to classes will also result in the need to migrate the tests - although that will be an issue in any case.

that was my same concern. 
however in this case probably we won't use this object in many tests so probably not a big issue.